nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-processor.js

```diff
@@ -1140,9 +1188,9 @@
          */
         get: function (id) {
             if (nfCommon.isUndefined(id)) {
-                return processorMap.values();
+                return Array.from(processorMap.values());
             } else {
-                return processorMap.get(id);
+                return processoMap.get(id);
```

Comment:
```suggestion
                return processorMap.get(id);
```

This typo currently breaks updating processor configurations.

Reply:
You found the misplaced 'r'!